.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["@babel/preset-env", "@babel/preset-react", "airbnb"]
+}
```

Comment:
Need an extra line break at the end here. The AirBnB style guide usually points this out.